lightkurve/targetpixelfile.py

```diff
@@ -61,6 +168,115 @@ def header(self, ext=0):
         """Returns the header for a given extension."""
         return self.hdu[ext].header
 
+    def get_prf_model(self):
```

Comment:
`wcs` and `get_prf_model` are nothing to do with this feature. Why have these moved? Is this a problem with commit history?

Reply:
Yep! I merged the updated master branch with find_stars branch.